stock_picking_partner_language/picking_report.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+    <data>
+
+        <report 
```

Comment:
Ah, OK, I see. You're not overwriting but creating another one. Totally agree then.